XML/TestCases/FunctionalTests-FPGA.xml

```diff
@@ -0,0 +1,15 @@
+<TestCases>
+    <test>
+        <testName>XILINX-CARD-VALIDATION</testName>
+        <testScript>fpga-card-validation.sh</testScript>
+        <setupType>OneVM</setupType>
+        <OverrideVMSize>Standard_NP10s</OverrideVMSize>
+        <files>.\Testscripts\Linux\fpga-card-validation.sh,.\Testscripts\Linux\utils.sh</files>
+        <Platform>Azure</Platform>
+        <Category>Functional</Category>
+        <Area>FPGA</Area>
+        <Tags>fpga</Tags>
+        <Timeout>600</Timeout>
```

Comment:
Is it used inside of test script? If needed, please use other tag.

<TestParameters>
            <param>Timeout=600</param>
</TestParameters>

Reply:
Running `xbutil validate` on a system with multiple FPGAs may take a long time to finish. I checked other test cases, such as `TestCases/FunctionalTests-NVME.xml` and I thought this is the place that I can specify how many seconds the LISAv2 framework should wait to run the FPGA test case.

Reply:
https://github.com/LIS/LISAv2/blob/master/XML/TestCases/FunctionalTests-InfiniBand.xml#L29 This is the example how you define the test parameter. You can use another example, https://github.com/LIS/LISAv2/blob/master/XML/TestCases/FunctionalTests-InfiniBand.xml#L35. The second case is more flexible to change the values in the run-time, not edit the test script.

Reply:
@LiliDeng mentioned this timeout setting is ok, and the default is 5 min. I am not sure if 600 is seconds or minutes.